api/v1alpha1/cluster_types.go

```diff
@@ -50,6 +52,13 @@ type CrdbClusterSpec struct {
 	// (Optional) The secret with a certificate and a private key for root database user
 	// Default: ""
 	ClientTLSSecret string `json:"clientTLSSecret,omitempty"`
+	// TODO document on how you need to set the max or the min
+	// The maximum number of pods that can be unavailable during a rolling update.
+	// This number is set in the PodDistruptionBudget and defaults to 1.
+	MaxUnavailable *int32 `json:"maxUnavailable,omitempty"`
+	// The min number of pods that can be unavailable during a rolling update.
```

Comment:
The min number of pods that will be available during a rolling update?

The description and variable name seem to diverge.

Reply:
It is a typo :)